examples/benzeneselenol.cjson

```diff
@@ -0,0 +1,279 @@
+                          [ 0.    0.   -0.03]
+                          [-0.01 -0.07  0.97]]
+
+                       [  [ 0.    0.01  0.19]
+                          [ 0.    0.    0.03]
+                          [ 0.   -0.01 -0.2 ]
+                          ...,
+                          [-0.02  0.    0.11]
+                          [ 0.    0.    0.04]
+                          [-0.04 -0.04  0.73]]
+
+                       [  [ 0.03 -0.11  0.01]
+                          [-0.13 -0.01  0.  ]
+                          [-0.12  0.2  -0.01]
+                          ...,
+                          [ 0.32 -0.09  0.  ]
+                          [-0.03 -0.08  0.  ]
+                          [ 0.37 -0.14  0.15]]
+
+                          ...,
+                       [  [-0.04  0.    0.  ]
+                          [-0.01  0.02  0.  ]
+                          [-0.01 -0.03  0.  ]
+                          ...,
+                          [-0.24 -0.43  0.  ]
+                          [ 0.    0.    0.  ]
+                          [ 0.    0.    0.  ]]
+
+                       [  [-0.02  0.    0.  ]
+                          [ 0.02 -0.03  0.  ]
+                          [-0.03 -0.06  0.  ]
+                          ...,
+                          [ 0.21  0.37  0.  ]
+                          [ 0.    0.    0.  ]
+                          [ 0.    0.    0.  ]]
+
+                       [  [-0.06  0.    0.  ]
+                          [-0.02  0.04  0.  ]
+                          [ 0.02  0.02  0.  ]
+                          ...,
+                          [ 0.2   0.37  0.  ]
+                          [ 0.    0.    0.  ]
+                          [ 0.    0.    0.  ]]]
+    "intensities": [  3.1499	3.2259	0.3196	0.0386
+                      0.0307	9.8302	0.0949	10.3548
+                      14.2804	56.5224	7.7038	0.0192
+                      0.4721	2.6001	5.1609	0.0025
+                      13.5903	14.2523	3.9936	0.1947
+                      2.1434	1.1767	1.2111	7.4301
+                      26.7973	1.4953	37.3637	9.7197
+                      2.2009	0.4627	7.5344	18.0167
+                      11.2 ]
+  },
+  "optimization": {
+    "done": True
+    "status": [1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 2 2 ]
+    "nbasis": 407
+    "nmo": 405
+    "scfenergies": [ -71671.43702915 -71671.4524142  -71671.4534768  -71671.45447492
+                     -71671.4556548  -71671.45605671 -71671.43194906 -71671.45761021
+                     -71671.45850275 -71671.39630296 -71671.45915119 -71671.45935854
+                     -71671.4594614  -71671.45947338 -71671.45948807 -71671.4594946
+                     -71671.4594946 ]
+  },
+  "properties": {
+    "molecular mass": 157.07,
+    "charge": 0,
+    "enthalpy":-2633.77264,
+    "multiplicity": 1,
+    "number of atoms": 13,
+    "temperature": 298.15,
+    "orbitals": {
+      "homos": [37],
+      "energies":[ [ -12350.7903	-1580.40486	 -1418.61794
```

Comment:
This is an outdated diff, I have provided a new commit with ',' separation but I missed the double [[